tools/cli/internal/cli/merge/merge.go

```diff
@@ -29,14 +30,24 @@ const (
 )
 
 type Opts struct {
+	Merger        openapi.Merger
 	basePath      string
 	outputPath    string
 	externalPaths []string
 }
 
 func (o *Opts) Run(_ []string) error {
-	// To add in follow up PR: CLOUDP-225849
-	return o.saveFile([]byte("test"))
+	federated, err := o.Merger.MergeOpenAPISpecs(o.externalPaths)
```

Comment:
can we add unit tests or do we want to capture in another ticket?

Reply:
planning to add them in a follow-up PR where I add mocking